### PR TITLE
added postgresql-contrib to vagrant bootstrap.sh

### PR DESCRIPTION
**Pull request by luuduong**
added the postgresql-contrib package

### CODE REVIEW

File: bin/bootstrap.sh [hunk 1]
```diff
@@ -25,6 +25,7 @@ echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main"
 wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
 apt-get -y update
 apt-get install -y postgresql-9.4 libpq-dev
+apt-get install -y postgresql-contrib
```
Comment (mokhan) [inline bin/bootstrap.sh]: postgresql-9.4-contrib?

Comment (mokhan) [inline bin/bootstrap.sh]:
```bash
vagrant@rails-dev-box:/vagrant/rails$ apt-cache search postgres | grep postgresql-contrib
postgresql-contrib - additional facilities for PostgreSQL (supported version)
postgresql-contrib-9.1 - additional facilities for PostgreSQL
postgresql-contrib-8.4 - additional facilities for PostgreSQL
postgresql-contrib-8.2 - additional facilities for PostgreSQL
postgresql-contrib-8.3 - additional facilities for PostgreSQL
postgresql-contrib-9.0 - additional facilities for PostgreSQL
postgresql-contrib-9.2 - additional facilities for PostgreSQL
postgresql-contrib-9.3 - additional facilities for PostgreSQL
postgresql-contrib-9.4 - additional facilities for PostgreSQL
```

**Comment by luuduong** [comment]
I added the specific version postgresql-9.4-contrib, done.

**Comment by mokhan** [comment]
:ship:

**Comment by luuduong** [comment]
@mokhan notify

**Pull request merged**